guides/setup.md

```diff
@@ -0,0 +1,95 @@
+---
+title: DustJS | Setup
+layout: guides
+permalink: /guides/setup/
+---
+
+# Installing Dust
+
+## npm
+
+To install as a Node module:
+
+```
+npm install --save dustjs-linkedin
+```
+
+Dust should work with any version of Node or io.js newer than 0.8.12.
+
+The command-line compiler is located at `./node_modules/.bin/dustc`
```

Comment:
Someone who knows Node would know this, but I don't think we should assume that all of our users know Node. Does it hurt to add the comment?